SteamProject/MySteam/Game.cpp

```diff
@@ -16,52 +19,93 @@ FGame::FGame(const std::string NewGameName, const std::string NewStudioName, con
+    ReleaseDay = InGame.ReleaseDay;
+    ReleaseMonth = InGame.ReleaseMonth;
+    ReleaseYear = InGame.ReleaseYear;
+}
+
+//-------------------------------------------------------------------------------------------------------------
 
-//----------------------getters------------------------
-std::string FGame::GetGameName() const
+FGame::~FGame()
+{
+    //the destructor
+}
+
+/* -------------------------------------------------------------------------------------------------------------
+*                               Getters      
+* -------------------------------------------------------------------------------------------------------------*/
+const std::string& FGame::GetGameName() const
 {
     return GameName;
 }
 
-std::string FGame::GetStudioName() const
+//-------------------------------------------------------------------------------------------------------------
+const std::string& FGame::GetStudioName() const
 {
     return StudioName;
 }
 
-std::string FGame::GetReleaseDate() const
+//-------------------------------------------------------------------------------------------------------------
+const std::string FGame::GetReleaseDate() const
 {
     return std::to_string(ReleaseDay) + "/" + std::to_string(ReleaseMonth) + "/" + std::to_string(ReleaseYear);
 }
 
 
-//----------------------setters------------------------
-void FGame::SetGameName(const std::string NewGameName)
+/* -------------------------------------------------------------------------------------------------------------
+*                               Setters
+* -------------------------------------------------------------------------------------------------------------*/
+
+void FGame::SetGameName(const std::string& NewGameName)
 {
     GameName = NewGameName;
 }
 
-void FGame::SetStudioName(const std::string NewStudioName)
+//-------------------------------------------------------------------------------------------------------------
+
+void FGame::SetStudioName(const std::string& NewStudioName)
 {
     StudioName = NewStudioName;
 }
 
-void FGame::SetReleaseDay(const int NewReleaseDay)
+//-------------------------------------------------------------------------------------------------------------
+
+void FGame::SetReleaseDay(const int& NewReleaseDay)
 {
     ReleaseDay = NewReleaseDay;
 }
 
-void FGame::SetReleaseMonth(const int NewReleaseMonth)
+//-------------------------------------------------------------------------------------------------------------
+
+void FGame::SetReleaseMonth(const int& NewReleaseMonth)
 {
     ReleaseMonth = NewReleaseMonth;
 }
 
-void FGame::SetReleaseYear(const int NewReleaseYear)
+//-------------------------------------------------------------------------------------------------------------
+
+void FGame::SetReleaseYear(const int& NewReleaseYear)
 {
     ReleaseYear = NewReleaseYear;
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
```

Comment:
Seeing as how you made bigger section headers for commenting on lines 61 - 63, this ends up being placed in the setter section. I think you can maybe add another section title here like "Utility Functions" or just move this before the getters and setters